content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,133 @@
+    --from-file=vm/ca-cert.pem \
+    --from-file=vm/ca-key.pem \
+    --from-file=vm/root-cert.pem \
+    --from-file=vm/cert-chain.pem
+$ cd ../..
+{{< /text >}}
+
+1. Generate a `cluster.env` configuration file that informs the virtual machine
+   deployment which network CIDR to capture and redirect to the Kubernetes
+   cluster:
+
+    {{< text bash >}}
+    $ ISTIO_SERVICE_CIDR=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
+    $ echo -n ISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR > $HOME/cluster.env
+    {{< /text >}}
+
+1. If the Kubernetes cluster makes workload requests to the virtual machine,
+   the ports used in the request must be specified in `cluster.env`. An
+   example of exposing the `3306` and `8080` ports on the virtual machine:
+
+    {{< text bash >}}
+    $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> $HOME/cluster.env
+    {{< /text >}}
+
+1. Install Istio with mesh expansion features:
+
+    {{< text bash >}}
+    $ istioctl install -f install/kubernetes/operator/examples/vm/values-istio-meshexpansion.yaml
+    {{< /text >}}
+
+### Installing the virtual machine
+
+Run the following commands on the virtual machine you want to add to the Istio mesh:
+
+1. Copy the previously created `cluster.env` and `*.pem` files to the virtual machine.
+
+1. Install the Debian package with the Envoy sidecar.
+
+    {{< text bash >}}
+    $ curl -LO https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb
+    $ sudo dpkg -i istio-sidecar.deb
+    {{< /text >}}
+
+1. Add an IP address for Istiod to `/etc/hosts`.  If using multiple networks,
+    replace `<ISTIOD_IP>`  with the ingress gateway service of istiod.
+    Revisit the instructions in the
+    [Determining the Ingress IP and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports). If using a single network, substitute `<ISTIOD_IP>` with the service
+    IP address of the `istiod` service.
+
+    {{< text bash >}}
+    $ sudo echo "<ISTIOD_IP> istiod.istio-system.svc" >> /etc/hosts
+    {{< /text >}}
+
+    {{< idea >}}
+    A sophisticated option involves configuring DNS within the virtual
+    machine to reference an external DNS server. This option is beyond
+    the scope of this document.
+    {{< /idea >}}
+
+1. Obtain `tcp-istiod` port for Istio ingress gateway and add this to `cluster.env`. This
+    defaults to 15012. If it has been customized, reference
+    [Determining the Ingress IP and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports) to learn how to obtain the `tcp-istiod` port.
+
+    {{< text bash >}}
+    $ echo "ISTIO_PILOT_PORT=15012" >> cluster.env
+    {{< /text >}}
+
+1. Install `root-cert.pem`, `key.pem` and `cert-chain.pem` within the directory `/etc/certs/`.
```

Comment:
where are the pem file coming from? have you tried to use the one from `samples\certs` directly?

Reply:
the pem files are coming from the `make` operation. I have tried this out in single network, and was not able to get this to work. I had hoped to try this in multicluster tonight - in the next few hours, then clean up the review, and mark it manually tested and done :)

Reply:
we need to copy `root-cert.pem`, `workload-cert-chain.pem` and `key.pem` and rename `workload-cert-chain.pem` to `cert-chain.pem` with the directory `/etc/certs/`.

Reply:
we need to copy root-cert.pem, workload-cert-chain.pem and key.pem and rename workload-cert-chain.pem to cert-chain.pem within the directory /etc/certs/.

Reply:
@irisdingbj can you verify the latest workflow makes sense as per your comments here.